File: lp1/lista1/10.py
n1 = int(input("Digite o primeiro número: "))
n2 = int(input("Digite o segundo número: "))

if n1 >= n2:
    maior = n1
    menor = n2
elif n2 > n1:
    maior = n2
    menor = n1

print(f"{maior}+{menor}={maior+menor}")
print(f"{maior}-{menor}={maior-menor}")
print(f"{maior}*{menor}={maior*menor}")
if menor == 0:
    print("Não é possível dividir por 0")
else:
    print(f"{maior}/{menor}={maior/menor}")

File: lp1/lista1/10.c
/*
Solicite dois valores inteiros ao usuário e mostre todos os valores inteiros,
iniciando no valor menor até o valor maior, inclusive.
*/
#include <stdio.h>

int main()
{
    int n1, n2, maior, menor;
    printf("Digite o primeiro valor: \n");
    scanf("%d", &n1);
    printf("Digite o segundo valor: \n");
    scanf("%d", &n2);
    if (n1 > n2)
    {
        maior = n1;
        menor = n2;
    }
    else
    {
        maior = n2;
        menor = n1;
    }
    for (int i = menor; i <= maior; i++)
    {
        printf("%d\n", i);
    }

    return 0;
}